boards/nxp/imx93_evk/imx93_evk_mimx9352_m33_ddr_defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CLOCK_CONTROL=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CONSOLE=y
+CONFIG_XIP=y
```

Comment:
What's this for? Isn't the memory supposed to be RW? Also, why the code/data split?

Reply:
I followed the structure from imx8mp m7 ddr support:
https://github.com/zephyrproject-rtos/zephyr/blob/2f23313e37e431d64972133e3c946a2829dad142/boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_m7_ddr_defconfig#L12

I tested for option CONFIG_XIP=y. Should we use CONFIG_XIP=n? As I understand from the https://github.com/zephyrproject-rtos/zephyr/blob/2f23313e37e431d64972133e3c946a2829dad142/include/zephyr/arch/arm/cortex_m/scripts/linker.ld#L22 the CONFIG_XIP=y creates the code/data split. That's why it's necessary to zephyr,flash and zephyr,sram in [boards/nxp/imx93_evk/imx93_evk_mimx9352_m33_ddr.dts](https://github.com/zephyrproject-rtos/zephyr/pull/79993/files/3d1349c4c1df78d4c2f777f60dffe7f979f913bb#diff-7d0ac3cd20b2794ee8e8c79f9404c7b838918b427138caba6143900ce56c1d4f)

Is CONFIG_XIP=y not the right way ?

Reply:
I'd suggest disabling it, don't see a reason why we'd want to use it with everything placed in DDR